docs/source/intro_to_nwb/4_apis.rst

```diff
@@ -0,0 +1,23 @@
+Software for reading and writing NWB data
+=========================================
+
+NWB provides two APIs for easily reading and writing NWB data: :pynwb-docs:`PyNWB <>` for Python and
+:matnwb-src:`MatNWB <>` for MATLAB.
+
+Each neurodata type defined in the NWB schema has a corresponding class in both PyNWB and MatNWB.
+Like neurodata types, these classes relate to each other through inheritance and composition.
+For example, the PyNWB class :py:class:`~pynwb.base.TimeSeries`
+serves as the base class (superclass) for more specialized time series classes, such as,
+:py:class:`~pynwb.ecephys.ElectricalSeries`, and a :py:class:`~pynwb.ecephys.LFP` object can contain one or more
+:py:class:`~pynwb.ecephys.ElectricalSeries` objects, just like how the corresponding neurodata types
+are defined in the NWB schema.
+
+To learn more about how to install the NWB APIs and use them to read and write NWB data, please
+see their respective documentation:
+
+* :pynwb-docs:`PyNWB <>` for Python
+* :matnwb-src:`MatNWB <>` for MATLAB
+
+To learn more about the NWB schema, please see the :nwb-schema-docs:`NWB Schema <>` documentation.
```

Comment:
I would also add a sentence along the lines of:

```To learn more about the broad range of core tools available for NWB see the :ref:`glossary-of-core-nwb-tools`_ and :ref:`analysis-and-visualization-tools`_ sections.```

I'm not sure if the reference are correct, but the intent is to refer to the sections of the nwb-overview where we have the NWB tool glossary and analysis tools listing.